test/e2e/frictionless/step-definitions/dc.steps.js

```diff
@@ -238,4 +251,59 @@ Then(/^I should (|not )see a modal promoting the browser extension$/, { timeout:
 
 Then(/^I dismiss the extension modal$/, async function () {
   await this.page.closeExtensionModal.click();
+});
+
+Then(/^I should be able to open the submenu of the (.*) menu item(?:|s)$/, async function (items) {
+  this.context(FrictionlessPage);
+  let menuItems = items.replace(/ and /g, ",").split(",");
+  menuItems = menuItems.map((x) => x.trim()).filter((x) => x.length > 0);
+  for (let item of menuItems) {
+    const index = cardinal(item);
+    await this.page.openSubMenu(index);
+    await expect(this.page.fedsPopup).toBeVisible();
+    await this.page.fedsPopup.screenshot({path: `fedsPopup-${item}.png`});
+    await this.page.closeSubMenu(index);
+    await expect(this.page.fedsPopup).not.toBeVisible();
+  }
+});
+
+Then(/^I should be able to use the "([^\"]*)" submenu$/, async function (menu) {
+  this.context(FrictionlessPage);
+  await this.page.openSubMenu(menu);
+  await expect(this.page.fedsPopup).toBeVisible();
+  await this.page.closeSubMenu(menu);
+  await expect(this.page.fedsPopup).not.toBeVisible();
+});
+
+Then(/^I select the last item of the submenu of the ([^\"]*) menu item$/, async function (menu) {
+  this.context(FrictionlessPage); 
+  await this.page.openSubMenu(cardinal(menu));
+  await this.page.selectFedsPopupItem(-1);
+});
+
+Then(/^I click "Buy now" button in the header$/, async function () {
+  this.context(FrictionlessPage);
+  this.page.buyNow.click()
```

Comment:
Hi @TsayAdobe,

I get this error when I run the test locally.
```
locator.click: Error: strict mode violation: locator('.feds-cta-wrapper a, .gnav-cta a') resolved to 2 elements:
           1) <a href="/acrobat/pricing.html" class="feds-cta fed…>Acrobat plans and pricing</a> aka getByText('Acrobat plans and pricing')
           2) <a daa-ll="Buy_now-8" class="feds-cta feds-cta--pri…>Buy now</a> aka getByRole('link', { name: 'Buy now' })
```

I think we want to target this element? `.feds-navItem > .feds-cta-wrapper a`

Reply:
Add `>> visible=true` to the selector